src/code/code_800EC960.c

```diff
@@ -1751,710 +1771,715 @@ void func_800EEA50(GfxPrint* printer) {
-            GfxPrint_SetPos(printer, D_80131F1C + 12, 6);
-            GfxPrint_Printf(printer, "%s", &digitStr);
+            digitStr[0] = (char)('0' + ((D_8013331C[sAudioSEParamChgWork[0]][sAudioSEParamChgWork[1]].unk_2 >>
+                                         (15 - sAudioSEParamChgBitSel)) &
+                                        1));
+            GfxPrint_SetPos(printer, 12 + sAudioSEParamChgBitSel, 6);
+            GfxPrint_Printf(printer, "%s", digitStr);
 
             SETCOL(255, 255, 255);
             GfxPrint_SetPos(printer, 3, 7);
-            GfxPrint_Printf(printer, "SE PR  : %02x", D_8013331C[D_80131F20[0]][D_80131F20[1]].unk_0);
+            GfxPrint_Printf(printer, "SE PR  : %02x",
+                            D_8013331C[sAudioSEParamChgWork[0]][sAudioSEParamChgWork[1]].unk_0);
             break;
 
-        case 14:
+        case PAGE_FREE_AREA:
             GfxPrint_SetPos(printer, 3, 4);
             SETCOL(255, 255, 255);
-            GfxPrint_Printf(printer, "env_fx %d code_fx %d SPEC %d", D_80130614, D_80130618, D_80133414);
+            GfxPrint_Printf(printer, "env_fx %d code_fx %d SPEC %d", sAudioEnvReverb, sAudioCodeReverb, gAudioSpecId);
 
-            if (sAudioUpdateTaskEnd == sAudioUpdateTaskStart) {
-                D_80131C90 = (f32)((sAudioUpdateEndTime - sAudioUpdateStartTime) * 64ULL / 3) / 50000000.0f;
-                if (D_80131C94 < D_80131C90) {
-                    D_80131C94 = D_80131C90;
+            if (sAudioUpdateTaskStart == sAudioUpdateTaskEnd) {
+                sAudioUpdateDuration = (f32)((sAudioUpdateEndTime - sAudioUpdateStartTime) * 64ULL / 3) / 50000000.0f;
+                if (sAudioUpdateDurationMax < sAudioUpdateDuration) {
+                    sAudioUpdateDurationMax = sAudioUpdateDuration;
                 }
             }
 
             GfxPrint_SetPos(printer, 3, 6);
-            GfxPrint_Printf(printer, "SOUND GAME FRAME NOW %f", D_80131C90);
+            GfxPrint_Printf(printer, "SOUND GAME FRAME NOW %f", sAudioUpdateDuration);
 
             GfxPrint_SetPos(printer, 3, 7);
-            GfxPrint_Printf(printer, "SOUND GAME FRAME MAX %f", D_80131C94);
+            GfxPrint_Printf(printer, "SOUND GAME FRAME MAX %f", sAudioUpdateDurationMax);
 
             GfxPrint_SetPos(printer, 3, 9);
             GfxPrint_Printf(printer, "SWITCH BGM MODE %d %d %d (FLAG %d)", D_8013061C, D_80130634, D_80130638,
                             D_80130654);
 
             GfxPrint_SetPos(printer, 3, 10);
-            GfxPrint_Printf(printer, "ENEMY DIST %f VOL %3d", D_80130620, D_80130624);
+            GfxPrint_Printf(printer, "ENEMY DIST %f VOL %3d", sAudioEnemyDist, sAudioEnemyVol);
 
             GfxPrint_SetPos(printer, 3, 11);
-            GfxPrint_Printf(printer, "GANON DIST VOL %3d", D_8016B8B4);
+            GfxPrint_Printf(printer, "GANON DIST VOL %3d", sAudioGanonDistVol);
```

Comment:
another reason to be wary of just using string names verbatim: what does this really mean? they sometimes refer to ganondorf as ganon (since thats the actor name). and looking at uses of this, I can only see it being touched by the river_sound actor, which afaik has nothing to do with ganon. unless the ganon fight uses river sound for something that I have never learned about before